snare/tests/test_get_body.py

```diff
@@ -50,7 +55,11 @@ def test_get_body(self):
         with open(os.path.join(self.target_path, self.hashname)) as f:
             self.return_content = f.read()
 
-        self.assertEqual(self.return_content, '<html><body><a href="/test"></a></body></html>')
+        self.expected_content = '<html><body><a href="/test"></a></body></html>'
+        self.assertEqual(self.return_content, self.expected_content)
+
+        self.assertEqual(self.handler.visited_urls[-1], 'http://example.com/test')
```

Comment:
I think you can check all the new additions.

Reply:
I think more than one URLs are added to `visited_urls`, so please check them too.

Reply:
Done.

Reply:
`self.handler.visited_urls[-2:]==['http://example.com/', 'http://example.com/test']` seems better.

Reply:
Sure.